Library/Homebrew/shims/super/cc

```diff
@@ -188,7 +188,7 @@ class Cmd
     when "-Xpreprocessor", "-Xclang"
       # used for -Xpreprocessor -fopenmp
       args << arg << enum.next
-    when /-mmacosx-version-min=10\.(\d+)/
+    when /-mmacosx-version-min=\d+\.(\d+)/
       arg = "-mmacosx-version-min=10.9" if high_sierra_or_later? && $1.to_i < 9
```

Comment:
Hmm sounds like this `if` condition will trigger for 11.0, which isn't what is meant to happen.

Reply:
I think it probably should happen there: it's bumping e.g. `-mmacosx-version-min=10.7` to `-mmacosx-version-min=10.9` on High Sierra or later. That seems correct on 11.0.

Reply:
I'm talking about how `-mmacosx-version-min=11.0` would also be changed to `-mmacosx-version-min=10.9`.

Reply:
Ah, yes, lol. Fixing.